aten/src/ATen/native/cuda/linalg/BatchLinearAlgebraLib.h

```diff
@@ -65,4 +65,18 @@ void lu_factor_looped_cusolver(const Tensor& self, const Tensor& pivots, const T
 
 #endif  // USE_CUSOLVER
 
+#if defined(BUILD_LAZY_CUDA_LINALG)
+namespace cuda { namespace detail {
+struct LinalgDispatch {
```

Comment:
nit: can you add a comment here that these are old-style dispatches and people shouldn't add to this list, and instead prefer implementing with stubs right away?